content/en/docs/Concepts/dependency-resolution/catalogsource-priority.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "CatalogSource Priority"
+weight: 2
+---
+
+As part of the SAT dependency resolver, OLM can choose from a wide range of catalogs to resolve dependencies. CatalogSource is a CR that defines where the catalog comes from and, therefore, can infer the amount of confidence cluster administrators or operator subscribers would want to rely on certain catalogs. OLM would like to explicitly weight catalogs for the dependency resolver so that certain catalogs can be priorities. The default CatalogSources such as Red Hat, Certified, and community catalogs will be assigned default priorities and subject to changes. 
+
+```yaml
```

Comment:
How useful is it to include this sample manifest?

Reply:
Im not sure how else to show the extra field. i mean it is better than say `.spec.priority` right.